src/community/libdeflate/src/main/java/org/geoserver/web/LibdeflateSettingsPanel.html

```diff
@@ -0,0 +1,47 @@
+<html xmlns:wicket="http://wicket.apache.org/">
+<head></head>
+<body>
+    <wicket:panel>
+        <legend>
+            <span>
+                <wicket:message key="Libdeflate.advancedSettingsTitle">Libdeflate Plugin Settings</wicket:message>
+            </span>
+        </legend>
+        <div wicket:id="container">
+        <ul>
+            <li>
+                <label for="compressionPriority" class="text">
+                    <wicket:message key="Libdeflate.compressionPriority">Compression priority</wicket:message>
+                </label>
+                <input id="compressionPriority" class="field text" type="text"
+                       wicket:id="compressionPriority" />
+            </li>
+
+            <li>
+                <label for="decompressionPriority" class="text">
+                    <wicket:message key="Libdeflate.decompressionPriority">Decompression priority</wicket:message>
+                </label>
+                <input id="decompressionPriority" class="field text" type="text"
+                       wicket:id="decompressionPriority" />
+            </li>
+
+     <li>
+         <label for="minLevel" class="text">
+             <wicket:message key="Libdeflate.minLevel">Minimum supported Compression Level</wicket:message>
+         </label>
+         <input id="minLevel" class="field text" type="text"
+                wicket:id="minLevel" />
+     </li>
+
+     <li>
+     <label for="maxLevel" class="text">
+     <wicket:message key="Libdeflate.maxLevel">Maximum supported Compression Level</wicket:message>
+     </label>
+     <input id="maxLevel" class="field text" type="text"
```

Comment:
This (and `minLevel`) would probably make more sense as spinbox.

Reply:
For the moment, I would leave it as is. I didn't manage to find a spinbox example in other pieces of GeoServer.
Can you please point me to a class in GeoServer using it? (I didn't find spinbox in the GeoServer code. I got the idea, finding a QT spinbox on Google but I'm not sure about the Wicket counterpart so if you already know some piece using that would be great and I can re-use it )